src/jit/codegencommon.cpp

```diff
@@ -618,6 +618,21 @@ regMaskTP Compiler::compHelperCallKillSet(CorInfoHelpFunc helper)
     }
 }
 
+//------------------------------------------------------------------------
+// compChangeLife: Compare the given "newLife" with last set of live variables and update
+//  codeGen "gcInfo", siScopes, "regSet" with the new variable's homes/liveness.
+//
+// Arguments:
+//    newLife - the new set of variables that are alive.
+//
+// Assumptions:
+//    The set of live variables reflects the result of only emitted code, it should not be considering the becoming
+//    live/dead of instructions that has not been emitted yet. This is used to ensure [) "VariableLiveRange"
```

Comment:
Same grammar note as before.

Also: `This is used to ensure [)` => `This is used to ensure intervals inclusive of the beginning and exclusive of the end`? Otherwise, `[)` looks like a typo.

Reply:
I agree, thanks!